app/models/host.rb

```diff
@@ -303,6 +303,14 @@ def validate_esx_host_connected_to_vc
     nil
   end
 
+  def validate_set_node_maintenance
+    validate_unsupported("Turn On Node Maintenance Mode")
```

Comment:
The string here should say something like "Maintenance Mode is not available". Same for `alidate_unset_node_maintenance()`.